src/librehatti/urls.py

```diff
@@ -1,57 +1,62 @@
+    url(r'^material_report', register.material_report, name='material_report'),
+    url(r'^lab_report', register.lab_report, name='lab_report'),
+    url(r'^suspense_register', register.suspense_register, name='suspense_register'),
+    url(r'^registered_users', register.registered_users, name='registered_users'),
+    url(r'^filter_sub_category/', reports_views.filter_sub_category, name='filter_sub_category'),
+    url(r'^bill/', prints_views.bill, name='bill'),
+    url(r'^suspense_bill/', prints_views.suspense_bill, name='suspense_bill'),
+    url(r'^quoted_bill/', prints_views.quoted_bill, name='quoted_bill'),
+    url(r'^tax/', prints_views.tax, name='tax'),
+    url(r'^bills/', include('librehatti.bills.urls', namespace='bills')),
+    url(r'^suspense/', include('librehatti.suspense.urls', namespace='suspense')),
     url(r'^generate_register/', GenerateRegister.as_view(), name='view_register'),
-    url(r'^history/','librehatti.reports.previous_history.history'),
-    url(r'^details/','librehatti.reports.previous_history.details'),
-    url(r'^proforma_details/','librehatti.reports.previous_history.proforma_details'),
+    url(r'^history/',previous_history.history, name='history'),
+    url(r'^details/',previous_history.details, name='details'),
+    url(r'^proforma_details/',previous_history.proforma_details, name='proforma_details'),
     url(r'^admin/doc/', include('django.contrib.admindocs.urls')),
     url(r'^admin/lookups/', include(ajax_select_urls)),
-    url(r'^voucher/', include('librehatti.voucher.urls')),
-    url(r'^receipt/', 'librehatti.prints.views.receipt'),
-    url(r'^tinymce/', include('tinymce.urls')),
-    url(r'^programmeletter/', 'librehatti.programmeletter.views.programmeletter'),
-    url(r'pending_clearance_register', 'librehatti.reports.register.pending_clearance_register'),
-    url(r'tada_register', 'librehatti.reports.register.tada_register'),
-    url(r'tada_othercharges_register', 'librehatti.reports.register.tada_othercharges_register'),
-    url(r'client_details_according_to_amount', 'librehatti.reports.register.client_details_according_to_amount'),
-) + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
\ No newline at end of file
+    url(r'^voucher/', include('librehatti.voucher.urls', namespace='voucher')),
+    url(r'^receipt/', prints_views.receipt, name ='receipt'),
+    url(r'^tinymce/', include('tinymce.urls', namespace='tinymce')),
+    url(r'^programmeletter/', programmeletter_views.programmeletter, name='programmeletter'),
+    url(r'pending_clearance_register', register.pending_clearance_register, name='pending_clearance_register'),
+    url(r'tada_register', register.tada_register, name='tada_register'),
+    url(r'tada_othercharges_register', register.tada_othercharges_register, name='tada_othercharges_register'),
+    url(r'client_details_according_to_amount', register.client_details_according_to_amount, name='client_details_according_to_amount')
```

Comment:
How would it know to pick up `register.client_details_according_to_amount` from `reports`?

Reply:
You have notice the line number 11 in the same file.
from .reports import register
It means from current directory reports pick module register and in python module is python file[special case]. So it will pick register.py from reports directory and then call function "client_details_according_to_amount"